measurements/models.py

```diff
@@ -121,43 +121,3 @@ class Software(Base):
     software_version = Column(String)
 
     reports = relationship("Report", back_populates="software")
-
-class ReportFile(Base):
```

Comment:
Note: according to `git grep` this class was unused in `master`.

Reply:
Correct. It was left lying around hence I zapped it.